between random: 101033032
between random: 101033032
between random: 101033029
between random: 101033032
between random: 101033032
between random: 15457
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [164,19,212,109,126,0,119,40]
between random: [98,186,251,144,189,28,252,80]
sed: can't read /usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_frame.ivy: No such file or directory
""
/usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_frame.ivy: line 1358: error: assumption failed
output directory: temp/1834
implementation directory: /home/chris/TVOQE_UPGRADE_27/quic/aioquic/
implementation command: python3 examples/http3_server.py --quic-log /home/chris/qlog_aiquic2 --certificate /home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_cert.pem --private-key /home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_key.pem  -v --host 127.0.0.1 --port 4443 -l /home/chris/secret.log
../quic_server_test_newcoid_rtp_error (0) ...
implementation command: ['python3', 'examples/http3_server.py', '--quic-log', '/home/chris/qlog_aiquic2', '--certificate', '/home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_cert.pem', '--private-key', '/home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_key.pem', '-v', '--host', '127.0.0.1', '--port', '4443', '-l', '/home/chris/secret.log']
quic_process pid: 21382
timeout 100 ./build/quic_server_test_newcoid_rtp_error seed=538 the_cid=0 server_cid=1 client_port=4987 client_port_alt=4988
client return code: 1
FAIL
error: 1 tests(s) failed
